Narrowing conversion error on Windows.

Hello,

I'm trying out the library on Windows and it seems that I couldn't get it to work properly. Every time I try to compile the `helloworld` example, I got the `error C2397: conversion from 'long' to 'uint64_t' requires a narrowing conversion` at the `BINLOG_INFO("Hello {}!", "World");` line.

However, when I compile the same code with WSL (GCC 8.3.0) everything works as expected, no warning at all. I also was able to build the `helloworld` example in the CMake generated project, and it ran just fine with the same VS.

I'm using Visual Studio 2019 on Windows 10.

I might be missing something obvious but I couldn't figure it out. Thank you so much for your time and help.

--- a/include/binlog/create_source_and_event.hpp
+++ b/include/binlog/create_source_and_event.hpp
@@ -47,7 +47,7 @@
     if (_binlog_sid_v == 0)                                                                  \
     {                                                                                        \
       _binlog_sid_v = writer.session().addEventSource(binlog::EventSource{                   \
-        0, severity, #category, __func__, __FILE__, __LINE__, MSERIALIZE_FIRST(__VA_ARGS__), /* NOLINT */ \
+        0, severity, #category, __func__, __FILE__, std::uint64_t(__LINE__), MSERIALIZE_FIRST(__VA_ARGS__), /* NOLINT */ \
         binlog::detail::concatenated_tags(__VA_ARGS__).data()                                /* NOLINT */ \
       });                                                                                    \
       _binlog_sid.store(_binlog_sid_v);                                                      \

